instauto/api/client.py

```diff
@@ -115,25 +115,47 @@ def _init_scheduler(self):
     def _grab_cookies(self) -> dict:
         return self._session.cookies.get_dict()
 
-    def save_to_disk(self, file_name: str, overwrite: Optional[bool] = None) -> bool:
+    def to_json(self) -> str:
         cookies = self._grab_cookies()
-        overwrite = overwrite or False
-
+        state_as_dict = self.state.__dict__
         try:
-            f = open(file_name, "w" if not overwrite else "w+")
-            state_as_dict = self.state.__dict__
-            try:
-                logged_in_data = state_as_dict.pop('logged_in_account_data').__dict__
-            except KeyError:
-                logged_in_data = {}
+            logged_in_data = state_as_dict.pop('logged_in_account_data').__dict__
+        except KeyError:
+            logged_in_data = {}
 
-            as_json = self._json_dumps({
+        return self._json_dumps({
                 'State': state_as_dict,
                 'IGProfile': self.ig_profile.__dict__,
                 'DeviceProfile': self.device_profile.__dict__,
                 'LoggedInAccountData': logged_in_data,
                 'session.cookies': cookies
             })
+    
+    @classmethod
+    def from_json(cls, jsondata: str) -> "ApiClient":
+        data = orjson.loads(jsondata)
```

Comment:
`orjson.loads` excepts bytes as input. `from_json` should either accept `bytes` and `str` as input and serialize as appropiate, or just accept string.